python/envs/ControlRL/ControlHVACEnv.py

```diff
@@ -131,44 +119,54 @@ def step(self, u):
         TOUT1_target=self.TOUT1_target
         u = np.clip(u, -self.max_in_change, self.max_in_change)
         self.last_u = u # for rendering      
-
-        #calculate the output
-        #TOUTZ1 = TOUTZ1 + u[0]*dt+3*dt*u[1]-4*dt*u[2]+5*dt*u[3]
-        #PUE= PUE-u[0]*dt+1.2*dt*u[1]-2*dt*u[2]+1*dt*u[3]
                 
         # Increase or decrease the 5 input values
         self.ins = self.ins+ u
         self.ins = np.clip(self.ins,self.min_Tin, self.max_Tin)
 
+        #calculate the output
         TOUTZ1 = self.temp_func(var=self.TZ1)
         PUE = self.temp_func(var=self.PUE)
 
         # get MSE
         MSE1 = (self.TOUT1_target-TOUTZ1)**2
 
         # get cost function
-        costs = (self.TOUT1_target-TOUTZ1)**2 + .001*(u[0]**2)
+        costs = (self.TOUT1_target-TOUTZ1)**2
         TOUTZ1 = np.clip(TOUTZ1,self.min_TZ, self.max_TZ)       
                 
-        MSE1_scaled = MSE1/(self.TOUT1_target)**2 # scale by target temp
-            
-        #done = ((MSE1_scaled <= self.MSE_thresh1) & (MSE2_scaled <= self.MSE_thresh2) & (MSE3_scaled <= self.MSE_thresh3))
-       
+        MSE1_scaled = MSE1/(self.TOUT1_target)**2 # scale by target temp                   
         done = ((MSE1_scaled <= self.MSE_thresh1) )
         done = bool(done)
 
-        TOUTZ2 = 0        
-        
+        TOUTZ2 = 0              
 
         self.state = np.array([TOUTZ1, TOUTZ2, PUE])
-        if cfg['print_output']==True:
-            #print(self.ins)
-            print ("  ",TOUTZ1,self.TOUT1_target,done,self.steps)
+        if cfg['log_output']==True:
+            if cfg['print_output_single']==True:
+                print ("  ",TOUTZ1,self.TOUT1_target)
+            self.TOUT_data.append(TOUTZ1)
+            self.Step_data.append(self.steps)
+        if done==True:
+            txt=str(self.TOUT_data)+","
+            self.data+=txt
+            if cfg['print_output_all']==True:
+                print(self.data)
+            if cfg['save_output']==True:
+                output_file=cfg['output_file']
+                with open(output_file,'a') as file:
+                    file.write(self.data) 
```

Comment:
When you can, try to avoid using python keywords as variable names. It's often common practice to append an underscore to avoid name collisions e.g. `file_`